airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -975,6 +977,13 @@ def should_validate_schema_fixture(self, inputs: BasicReadTestConfig, test_stric
         else:
             return inputs.validate_schema
 
+    @pytest.fixture(name="should_validate_stream_statuses")
+    def should_validate_stream_statuses_fixture(self, inputs: BasicReadTestConfig, test_strictness_level: Config.TestStrictnessLevel):
+        if not inputs.validate_stream_statuses and test_strictness_level is Config.TestStrictnessLevel.high:
+            pytest.fail("High strictness level error: validate_stream_statuses must be set to true in the basic read test configuration.")
+        else:
+            return inputs.validate_stream_statuses
```

Comment:
There is [a grooming note](https://github.com/airbytehq/airbyte/issues/33565#issuecomment-1863264152) that mentions that we should not apply this by default for community connectors. Right now, the default value for `validate_stream_statuses` is True. Does isn't that contradictory?

Reply:
fixed, this test will not run if `ql < 400`

Reply:
With the new change, does that mean that if we want to enable this for `ql < 400`, we just can't? It feels like we should be able to test this even if the connector is ql < 400

Reply:
in this case, do we need to set it to `False` as default, and force to `True` for certified (`ql > 400`) connectors?

Reply:
Can we set it to "None" so that we have three states:
* Not explicitly defined which means ql > 400 are enabled but not the rest
* False would fail for ql > 400 but skip for non-certified
* True would work for everyone

Reply:
Good point!
fixed.